Fix missing space between description and parameters in FilterCommand usage message

## Bug Description
The `filter` command displays a malformed usage message when an invalid command format is entered. There is a missing space between the description and the `Parameters:` label.

## Steps to Reproduce
1. Type `filter` in the command box without any arguments

## Current Behaviour
```
[ERROR] Invalid command format!
filter: Filters the current list of persons by the given parametersParameters: -interviewed (y/n/0/1)
```

## Expected Behaviour
```
[ERROR] Invalid command format!
filter: Filters the current list of persons by the given parameters.
Parameters: -interviewed (y/n/0/1)
```

## Root Cause
Missing newline or space between the description and `Parameters:` label in `MESSAGE_USAGE` in `FilterCommand.java`.

--- a/src/main/java/seedu/address/logic/commands/FilterCommand.java
+++ b/src/main/java/seedu/address/logic/commands/FilterCommand.java
@@ -17,7 +17,7 @@ public class FilterCommand extends PredicateCommand {
     public static final String COMMAND_WORD = "filter";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD
-        + ": Filters the current list of persons by the given parameters"
+        + ": Filters the current list of persons by the given parameters\n"
         + "Parameters: -interviewed (y/n/0/1)\n"
         + "Example: " + COMMAND_WORD + " -interviewed y";
 

